daffodil-cli/src/windows/Product.wxs

```diff
@@ -0,0 +1,376 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Github isn't showing the LICENSE.rtf, so I'll put the comment here.

I'd really like to avoid maintenence of a second License.rtf file that has very subtle differences from the main license file. How legible is an auto generated RTF file in the installer? Perhaps we could so soemthing like chang the license agreement page to use smaller font so things are less likely to wrap?

Reply:
Send me your 'unoconv' version again and I'll evaluate it in both scrollable text box and print form.

Reply:
I need you to decide on how we are going to convert to RTF.

Reply:
Done.